cmake-tool/simulate_scripts/simulate.py

```diff
@@ -80,23 +82,26 @@ def notice(message):
                                   qemu_gdbserver_command]
     qemu_simulate_command = " ".join(qemu_simulate_command_opts)
 
-    notice(qemu_simulate_command)
+    notice('QEMU command: ' + qemu_simulate_command)
 
     if qemu_gdbserver_command != "":
-        notice('waiting for GDB on port 1234...')
-
-    qemu_status = subprocess.call(qemu_simulate_command, shell=True)
-
-    if qemu_status != 0:
-        delay = 5  # in seconds
-        # Force a newline onto the output stream.
-        sys.stderr.write('\n')
-        msg = "QEMU failed; resetting terminal in {d} seconds".format(d=delay) \
-            + "--interrupt to abort\n"
-        notice(msg)
-    else:
-        delay = 2  # in seconds
-
-    time.sleep(delay)
-
-    subprocess.call("tput reset", shell=True)
+        notice('GDB command: ' + qemu_gdbserver_command)
```

Comment:
Why are these prints changed too? `qemu_gdbserver_command` is already part of qemu command I think?

Reply:
Yes you're right, my bad.

Reply:
(I've discarded that change)